src/Bridge/Doctrine/Common/Filter/OrderFilterTrait.php

```diff
@@ -52,6 +52,13 @@ public function getDescription(string $resourceClass): array
                 'property' => $propertyName,
                 'type' => 'string',
```

Comment:
Actually, I think that we should deprecate `type` and `required`, in favor of `schema` (that is a JSON Schema). Using a JSON Schema to describe the filter is smart, and we should only support that. WDYT @teohhanhui?

Reply:
I think that'll just make a lot of code more complicated...